tools/src/logger.ts

```diff
@@ -0,0 +1,25 @@
+export type LogLevel = 'info' | 'warn' | 'error'
+
+export class Logger {
+  level: string
```

Comment:
Turn level into an integer/enum where info > warn > debug > trace and compare numbers. So if logger level is info, you also want to see warnings.